scripts/import_from_ena_legacy.py

```diff
@@ -0,0 +1,849 @@
+    "WGS": "WGS",
+    "Whole Genome Shotgun Sequence": "WGS",
+
+    "ChIP-Seq": "ChIP-Seq",
+    "ChIP-seq": "ChIP-Seq",
+    "ChIP-seq Histones": "ChIP-Seq",
+
+    "Hi-C": "Hi-C",
+
+    "ATAC-seq": "ATAC-seq",
+
+    "RNA-Seq": "RNA-Seq",
+    "RNA seq": "RNA-Seq",
+    "miRNA-Seq": "RNA-Seq",
+    "ssRNA-seq": "RNA-Seq",
+    "strand-specific RNA sequencing": "RNA-Seq",
+    "Transcriptome profiling": "RNA-Seq",
+    "RNA sequencing": "RNA-Seq",
+
+    "Bisulfite-Seq": "BS-Seq",
+    "Bisulfite Sequencing": "BS-Seq",
+    "BS-Seq": "BS-Seq",
+    "Whole Genome Bisulfite Sequencing": "BS-Seq",
+    "WGBS": "BS-Seq",
+    "Reduced Representation Bisulfite Sequencing": "BS-Seq",
+    "RRBS": "BS-Seq",
+
+    "DNase": "DNase",
+
+    "MiSeq": "Other",
+    "GeneChip": "Other",
+    "MeDIP-Seq": "Other",
+    "MeDIP": "Other",
+    "methylated DNA immunoprecipitation-sequencing": "Other",
+    "RIP-Seq": "Other"
+}
+
+SPECIES_DICT = {
+  "9031": "Gallus gallus",
+  "9913": "Bos taurus",
+  "9823": "Sus scrofa",
+  "9940": "Ovis aries",
+  "9796": "Equus caballus",
+  "9925":"Capra hircus"
+}
+SPECIES_TAXOMONY_LIST = list(SPECIES_DICT.keys())
+
+DATA_SOURCES = ['fastq', 'sra', 'cram_index']
+DATA_TYPES = ['ftp', 'galaxy', 'aspera']
+
+# holds all sample records from ES
+BIOSAMPLES_RECORDS = dict()
+# holds the material information for all records encountered (retrieve_biosample_record function)
+# keys are biosamples accessions and values are dicts which have three sub keys
+# confirmed (boolean), material (dict), accession (str)
+CACHED_MATERIAL = dict()
+
+ASSAY_TYPES_TO_BE_IMPORTED = {
+    "ATAC-seq": "ATAC-seq",
+    "BS-Seq": "methylation profiling by high throughput sequencing",
+    "Hi-C": "Hi-C",
+    "DNase": "DNase-Hypersensitivity seq",
+    # "RNA-Seq" => "",
+    "WGS": "whole genome sequencing assay",
+    "ChIP-Seq": "ChIP-seq"
+}
+EXPERIMENT_TARGETS = {
+    "ATAC-seq": "open_chromatin_region",
+    "BS-Seq": "DNA methylation",
+    "Hi-C": "chromatin",
+    "DNase": "open_chromatin_region",
+    "RNA-Seq": "Unknown ",
+    "WGS": "input DNA",
+    "ChIP-Seq": "Unknown"
+}
+# value of all is not allowed in the general ena data portal, not like FAANG, so need to list fields
+# which we want to retrieve
+FIELD_LIST = [
+    'study_accession', 'secondary_study_accession', 'sample_accession', 'experiment_accession', 'run_accession',
+    'submission_accession', 'tax_id', 'instrument_platform', 'instrument_model', 'library_strategy',
+    'library_selection', 'read_count', 'base_count', 'first_public', 'last_updated',
+    'study_title', 'study_alias', 'run_alias', 'fastq_bytes', 'fastq_md5',
+    'fastq_ftp', 'fastq_aspera', 'fastq_galaxy', 'submitted_format', 'sra_bytes',
+    'sra_md5', 'sra_ftp', 'sra_aspera', 'sra_galaxy', 'cram_index_ftp',
+    'cram_index_aspera', 'cram_index_galaxy', 'project_name'
+]
+
+# logger = utils.create_logging_instance('import_ena_legacy', level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
logger instance should be created script entry point (if __name__ == "__main__")

Reply:
done in 27a573a